Sources/EpoxyLayoutGroups/Groups/Group.swift

```diff
@@ -98,10 +98,12 @@ extension Constrainable where Self: InternalGroup {
     return items[index]
   }
 
-  /// Shared implementation of `setItems(_ newItems:)`
-  func _setItems(_ newItems: [GroupItemModeling], animated: Bool) {
+  /// Shared implementation of `_setItems` is used across different groups
+  /// - Parameters:
+  ///   - newItems: the new set of items to set on the group
+  ///   - animation: enumeration to animate or not, with `LayoutGroupUpdateAnimation` available for animation
+  func _setItems(_ newItems: [GroupItemModeling], animationStyle: GroupAnimationStyle) {
```

Comment:
I think we still want the `animated: Bool` flag passed to this method, and we would just pass an additional `animationStyle` that would decide whether it was animated or not that comes from the `Group.animation`